commands/Invite.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  name: "invite",
+  code: `$title[Music Bot invite links]
+  $description[
+   Recommended: [Click here]($getBotInvite)
+   Admin: [Click here](https://discord.com/api/oauth2/authorize?client_id=BOT_ID&permissions=8&scope=bot)
+  $footer[Thanks for Choosing Me]`
```

Comment:
This is not a BDFD bot, so something like this doesn't work.

you need an execute function
```suggestion
  execute(message, args, client) {
    message.channel.send(new MessageEmbed({
      title: "Music Bot invite links",
      description: stripIndents`
        Recommended: [Click here](${client.generateInvite(YOUR_PERMISSIONS_HERE)})
        Admin: [Click here](${client.generateInvite(Permissions.FLAGS.ADMINISTRATOR)})
      `,
      footer: "Thanks for choosing me"
    }));
  }
```